index.bs

```diff
@@ -1689,13 +1685,12 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
         :   If the user exercises a user agent user-interface option to cancel the process,
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
-            and [=set/remove=] |authenticator| from |issuedRequests|. Return a {{DOMException}} whose name is "{{NotAllowedError}}".
+            and [=set/remove=] |authenticator| from |issuedRequests|. Throw a "{{NotAllowedError}}" {{DOMException}}.
 
-        :   If the <code>|options|.{{CredentialCreationOptions/signal}}</code> is present and its
-            [=AbortSignal/aborted flag=] is set to [TRUE],
+        :   If the <code>|options|.{{CredentialCreationOptions/signal}}</code> is present and [=AbortSignal/aborted=],
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=]
-            operation on |authenticator| and [=set/remove=] |authenticator| from |issuedRequests|. Then return a {{DOMException}}
-            whose name is "{{AbortError}}" and terminate this algorithm.
+            operation on |authenticator| and [=set/remove=] |authenticator| from |issuedRequests|. Then throw the
+            <code>|options|.{{CredentialCreationOptions/signal}}</code>'s [=AbortSignal/abort reason=].
```

Comment:
I'm not sure if it was a typo, but I don't think it makes sense to change CredentialCreationOptions to CredentialRequestOptions here since the if condition just preceding this looks at `{{CredentialCreationOptions/signal}}`.

Also, I guess it was formulated as "the signal member" below because they weren't using the `options.` prefix, so it seems strange to change it to "the options.signal member". I changed "the signal member" to "options.signal" below as well. Hope that makes sense :)

Reply:
> I'm not sure if it was a typo, but I don't think it makes sense to change CredentialCreationOptions to CredentialRequestOptions here

Ah yes, sorry, that was a copy-paste error by me. Good catch, thanks!